screens/visualization/sketch.py

```diff
@@ -133,19 +149,18 @@ def __call__(self):
     ns_vel = offset(rotate(offset(arrow(arrow_size), (0., 1.4)),
                            225.*u.degree), ns_offset)
 
-    if figures == '9' or figures == ['9']:
-        screen_x = ns_offset[0] / 1.1
-    else:
-        screen_x = ns_offset[0] / 1.7
-    screen_y = [2.3, -2.3, -5.8, 4.2]
+    screen_x = ns_offset[0] / 1.7

```

Comment:
How about passing in `screen_x`, or perhaps rather `beta` where that has the usual definition of `beta = 1 - screen_x / d_psr`? Default can then be `1-1/1.7`.

Reply:
I passed beta. I thought this parameter is often called 's' in the literature. Is 'beta' more usual?